lib/tests/streamlit/elements/number_input_test.py

```diff
@@ -69,6 +70,32 @@ def test_just_disabled(self):
         c = self.get_delta_from_queue().new_element.number_input
         self.assertEqual(c.disabled, True)
 
+    def test_none_value(self):
```

Comment:
These tests could use the testing framework, though for examples this small it might not be any easier/clearer

Reply:
Good point; I will have a look into this once I have finished the `element_tree` implementation. This would look similar to the slider test here or?: https://github.com/streamlit/streamlit/blob/cc5520ad6c285473255082d6a742ab28e51decca/lib/tests/streamlit/elements/slider_test.py#L306